pkg/autoscaler/multiscaler_test.go

```diff
@@ -52,29 +52,31 @@ func watchFunc(ctx context.Context, ms *MultiScaler, metric *Metric, desiredScal
 			errCh <- fmt.Errorf("Get() = %v, wanted %v", got, want)
 			return
 		}
-		doneCh <- struct{}{}
+		errCh <- nil
 	}
 }
 
 // verifyTick verifies that we get a tick in a certain amount of time.
-func verifyTick(doneCh chan struct{}, errCh chan error) error {
+func verifyTick(errCh chan error) error {
 	select {
 	case err := <-errCh:
-		return err
-	case <-doneCh:
-		// We got the signal!
+		if err != nil {
```

Comment:
no need to check here - can just return err

Reply:
Fair enough! Fixed.